qml/Components/Dialogs.qml

```diff
@@ -243,9 +245,11 @@ MouseArea {
                 focus: true
                 text: i18n.ctr("Button: Power off the system", "Power off")
                 onClicked: {
-                    root.closeAllApps();
-                    powerDialog.hide();
-                    root.powerOffClicked();
+                    topLevelSurfaceList.closeAllWindows();
+                    doOnClosedAllWindows = function() {
+                        powerDialog.hide();
+                        root.powerOffClicked();
```

Comment:
Please order these calls consistently for each of these `doOnClosedAllWindows` blocks.

Reply:
These are left how the order they was in.